internal/admission/validator.go

```diff
@@ -87,9 +88,14 @@ func (validator KongHTTPValidator) ValidatePlugin(ctx context.Context,
 	if len(k8sPlugin.Protocols) > 0 {
 		plugin.Protocols = kong.StringSlice(k8sPlugin.Protocols...)
 	}
-	isValid, err := validator.PluginSvc.Validate(ctx, &plugin)
+	isValid, msg, err := validator.PluginSvc.Validate(ctx, &plugin)
 	if err != nil {
-		return false, ErrTextPluginConfigViolatesSchema, err
+		return false, ErrTextPluginConfigValidationFailed, err
+	}
+	// this doesn't work yet, since Validate still returns an "error" for informational stuff
+	// fixing that requires a breaking go-kong change
```

Comment:
Is there some follow up issue for this we can document here?

Reply:
This was a draft note I forgot to remove after actually finishing up the go-kong release with the changed needed to make this work.